test/specs/print_utils/tests/01.print_utils.js

```diff
@@ -2,6 +2,7 @@ exports.execute = function (options) {
     var module = options.includes.ermRest;
     var formatUtils = module._formatUtils;
     describe("Print utils, For pretty printing values based on a value's type, ", function () {
+        var catalog = {id: process.env.DEFAULT_CATALOG};
```

Comment:
I know currently you are only using `id`, but it's better if you pass the actual catalog object. You can access the current catalog using `options.catalog`.

Reply:
Or you can even change the catalog parameter of those functions to be optional. So if you pass the catalog we're going to attach `$catalog` and if you don't, we're not. I think this might be the better solution. You just need to add a check to `_addErmrestVarsToTemplate` function to make sure catalog is defined and then use it.